### ISSUE TITLE
Add option to preserve schema case in id 

### ISSUE THREAD

**Issue by DavidBiesack** [Mermade/widdershins#209]
widdershins normalizes each schema name to lowercase when generating ids. `/components/schemas/abcDef` results in

```
<h2 id="tocSabcdef">abcDef</h2>

<a id="schemaabcdef"></a>
```

Note: case is already preserved for `operationId`: `operationId: createResource` results in

```
## `createAccount`
<a id="opIdcreateAccount"></a>
```

**Describe the solution you'd like**

Add an option `options.preserveSchemaCase`. The default would be false, so as to not break existing deployments which have bookmarks in external doc to `#schemaabcdef`.

Preferable: add an `_` or other character before the schema name, to id strings are more legible

```
<h2 id="tocS_abcDef">abcDef</h2>

<a id="schema_abcDef"></a>
```

**Comment by MikeRalphson** [comment]
What benefit does this proposed change bring?

**Comment by ehallander9591** [comment]
I notice that the output examples don't accurately represent both attribute names, nor enum values. For example, depending on the language tab, but let's take javascript, proto_buf attribute some_field will generate openAPI some_field (swagger) gets turned into somefield in examples where generated javascript will be someField. Underscores preserved in enums are removed in examples. SOME_VALUE turns into SOMEVALUE. 

In the same generated markdown file I have both:
'javascript--nodejs as json (Wrong)
 "solvercriteriatype": "SCTNOTSPECIFIED", 

and
Body parameter as json (Correct
[sample.zip](https://github.com/Mermade/widdershins/files/2936876/sample.zip)

)
"solver_criteria_type": "SCT_NOT_SPECIFIED"

I dropped a sample

**Comment by MikeRalphson** [comment]
@ehallander9591 The language tab settings in your YAML front-matter look wrong:

```yaml
---
title: davinci/service/route/grpc/Routing.proto
language_tabs:
  - '''go': Go'
  - '''javascript--nodejs': Node.JS'
  - '''http': HTTP'
  - '''javascript': JavaScript'
toc_footers: []
includes: []
search: false
highlight_theme: darkula
headingLevel: 2

---
```

There are extra single quotation marks which should not be there. Can you try removing them and see what happens?

**Comment by MikeRalphson** [comment]
Should be fixed in version `4.0.0`, which includes breaking changes (please see the `README.md` file).
